Perl/Splitter/Makefile.PL

```diff
@@ -27,18 +27,7 @@ our $OPTIMIZE;
 #     #define MINIZ_HAS_64BIT_REGISTERS 1
 
 my $defines = join " ", map "-D$_", grep exists $ENV{$_}, qw(NOINLINE DEBUG MEMDEBUG NDEBUG);
-if ($Config{gccversion}) {
-    $OPTIMIZE = '-O3 -Wall -W';
```

Comment:
This is the only thing that I have intentionally dropped. The other makefiles only set `-O3`. I can reinstate or make `-Wall -W` the default - was just having a hard time coming up with a name for the option :)